greenhouse/metrics-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: bazel-cache-metrics
+  namespace: default
+spec:
+  selector:
+    app: bazel-cache
+  ports:
+  - name: default
+    protocol: TCP
+    port: 80
+    targetPort: 9090
+  loadBalancerIP: 35.225.115.154
```

Comment:
I don't think this need to be specified.

Reply:
later on velodrome need to poll the metrics from this IP though

Reply:
it does, this static IP is reserved for this use case, there are other static IPs for other things (boskos) so we need to tell the LB which one we want. (and also to use a static IP so we can point velodrome at it)